clarifai/client/app.py

```diff
@@ -288,6 +288,42 @@ def list_concepts(self, page_no: int = None,
       concept_info['id'] = concept_info.pop('concept_id')
       yield Concept(**concept_info)
 
+  def list_concept_relations(self,
+                             concept_id: str = None,
+                             predicate: str = None,
```

Comment:
```suggestion
                             predicate: str = '',
```

Reply:
Both None or empty string are working to get all the concept relations of a given concept

Reply:
Okay, then we can just make it optional, letting the users know that by default it returns all the relations.

Reply:
In this case add logging, if the request fails, We should return the error logging message in case user gives a different predicate in the request.

Reply:
Logging already present in the lister class

Reply:
Great then everything looks fine.